Content.Client/Overlays/ShowSecurityIconsSystem.cs

```diff
@@ -32,12 +34,20 @@ private void OnGetStatusIconsEvent(EntityUid uid, StatusIconComponent _, ref Get
             return;
         }
 
-        var securityIcons = DecideSecurityIcon(uid);
+        @event.StatusIcons.AddRange(GetSecurityIcons(uid));
 
-        @event.StatusIcons.AddRange(securityIcons);
+        var canDisplayEv = new CanDisplayStatusIconsEvent(_player.LocalSession?.AttachedEntity);
+        RaiseLocalEvent(ref canDisplayEv);
+
+        if (canDisplayEv.Cancelled)
+        {
+            return;
+        }
+
+        @event.StatusIcons.AddRange(GetCancellableSecurityIcons(uid));
```

Comment:
this API feels very awkward. i think that sechud icons should just be managed by another component so that borgs can simply have that omitted from their prototypes. This cancellable logic is very special-cased and causes weird dependencies between borgs and sechuds.